test/e2e/tests/test_alias.py

```diff
@@ -80,18 +80,19 @@ def lambda_function():
 class TestAlias:
     def test_smoke(self, lambda_client, lambda_function):
         (_, function_resource) = lambda_function
+        function_resource_name = function_resource["metadata"]["name"]
         lambda_function_name = function_resource["spec"]["name"]
 
         resource_name = random_suffix_name("lambda-alias", 24)
 
         replacements = REPLACEMENT_VALUES.copy()
         replacements["AWS_REGION"] = get_region()
         replacements["ALIAS_NAME"] = resource_name
-        replacements["FUNCTION_NAME"] = lambda_function_name
+        replacements["FUNCTION_REF_NAME"] = function_resource_name
         replacements["FUNCTION_VERSION"] = "$LATEST"
 
         # Load alias CR
-        resource_data = load_lambda_resource(
+        resource_data = load_lambda_resource( 
```

Comment:
looks like an extraneous whitespace found its way in here :)

Reply:
Oops! corrected it. Thanks.